yt_dlp/extractor/svt.py

```diff
@@ -472,7 +472,7 @@ def _real_extract(self, url):
         title = self._og_search_title(webpage)
 
         urql_state = self._search_json(
-            r'window\.svt\.nyh\.urqlState\s*=', webpage, 'json data', display_id)
+            r'window\.svt\.urqlState\s*=', webpage, 'json data', display_id)
```

Comment:
would it hurt to keep matching the old way as well? or are you positive that all pages now only use the new var?
```suggestion
            r'window\.svt\.(?:nyh\.)?urqlState\s*=', webpage, 'json data', display_id)
```

Reply:
it wouldn't hurt
> are you positive that all pages now only use the new var?
I'm not sure that all pages now uses the new var (I checked with new and old pages - they were using the new var).